Exiled.Events/EventArgs/Player/ActivatingGeneratorEventArgs.cs

```diff
@@ -14,7 +14,7 @@ namespace Exiled.Events.EventArgs.Player
     using MapGeneration.Distributors;
 
     /// <summary>
-    ///     Contains all information before a player filps the switch of the generator.
+    ///     Contains all information before a player filps the switch to a generator.
```

Comment:
I think the previous version was more correct

Reply:
This change is grammatically more correct, the switch is possive to the generator, so switch to a generator is more accurate. You wouldn't say "I flipped the switch of the generator" you would say "I flipped a switch to a generator"